components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/BuildCmd.java

```diff
@@ -117,6 +119,17 @@ public void execute() {
         }
     }
 
+    private boolean checkFolderContentAvailablity(String fileLocation) {
+        File file = new File(fileLocation);
+        FilenameFilter filter = (f, name) -> (name.endsWith(".yaml") || name.endsWith(".json"));
```

Comment:
Shall we add yml to the list. also we'll need to add support for .yml loading during the actual build as we discussed recently.